pjp.tm11/src/main/java/pjp/tm11/xyz/subho/login/controller/UserRegistrationServlet.java

```diff
@@ -0,0 +1,65 @@
+package pjp.tm11.xyz.subho.login.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import pjp.tm11.xyz.subho.login.bean.UserBean;
+import pjp.tm11.xyz.subho.login.dao.UserRegistrationDAO;
+
+/**
+ * Servlet implementation class UserRegistrationServlet
+ */
+public class UserRegistrationServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+       
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public UserRegistrationServlet() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5662373)